Src/library/memory_analyzer/object_graph/ma_reference_link.e

```diff
@@ -105,43 +105,50 @@ feature {EG_FIGURE, EG_FIGURE_WORLD} -- Update
 			an_angle: DOUBLE
 			source_size: EV_RECTANGLE
 			l_model: like model
+			l_source: like source
+			l_target: like target
 		do
 			l_model := model
-			check attached l_model end -- FIXME: Implied by ...?
-			if not l_model.is_reflexive then
-				if attached source as l_source and then attached target as l_target then
-					p1 := line.point_array.item (0)
-					p2 := line.point_array.item (1)
-
-					p1.set (l_source.port_x, l_source.port_y)
-					p2.set (l_target.port_x, l_target.port_y)
-
-					an_angle := line_angle (p1.x_precise, p1.y_precise, p2.x_precise, p2.y_precise)
-					l_source.update_edge_point (p1, an_angle)
-					an_angle := pi + an_angle
-					l_target.update_edge_point (p2, an_angle)
-				elseif attached source as l_source_2 then
-					p1 := line.point_array.item (0)
-					p1.set (l_source_2.port_x, l_source_2.port_y)
-					l_source_2.update_edge_point (p1, 0)
-				elseif attached target as l_target_2 then
-					p2 := line.point_array.item (1)
-					p2.set (l_target_2.port_x, l_target_2.port_y)
-					l_target_2.update_edge_point (p2, 0)
-				end
-
-				line.invalidate
-				line.center_invalidate
-				if is_label_shown then
-					name_label.set_point_position (line.x, line.y)
-				end
+			if l_model = Void then
```

Comment:
done.
Note that I initially did that, since it is sometime a pain with big if then else end code .. to see the associated condition for the "else" code.
with ` = Void ` we see it right away, and the else is not far from the `if`.

however I understand that here ... the `check ... end ` should never be reached so it sounds fair to put it as "else".